packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -350,3 +359,56 @@ function EmptyTest() {
     </div>
   );
 }
+
+
+export function DragExample(props?) {
+  let getItems = (keys) => [...keys].map(key => {
+    let item = items.find(item => item.key === key);
+    return {
+      'text/plain': item.textValue
+    };
+  });
+
+  let itemAllowsDragging = (key) => {
+    let item = items.find(item => item.key === key);
+    return item.isDraggable;
+  };
+
+  return (
+    <ListView
+      aria-label="draggable list view"
+      width="300px"
+      selectionMode="multiple"
+      items={items}
+      getItems={getItems}
+      disabledKeys={['f']}
+      itemAllowsDragging={itemAllowsDragging}
+      onDragStart={action('dragStart')}
+      onDragEnd={action('dragEnd')}
+      {...props}>
+      {(item: any) => (
+        <Item key={item.key} textValue={item.textValue}>
+          <Content>
+            <Flex alignItems="center" gap="10px">
+              <View flexGrow={1}>Item {item.key}</View>
+              <ActionButton><Add /></ActionButton>
+              <MenuTrigger>
+                <ActionButton><MoreSmall /></ActionButton>
+                <Menu>
+                  <Item>
+                    <Edit />
+                    <Text>Edit</Text>
+                  </Item>
+                  <Item>
+                    <Delete />
+                    <Text>Delete</Text>
+                  </Item>
+                </Menu>
+              </MenuTrigger>
```

Comment:
Another idea on how to fix the menu trigger not opening on click (original issue mentioned here: https://github.com/adobe/react-spectrum/pull/2593#discussion_r760678894):

Perhaps we could modify `usePress` to have `allowDragOnPress`. If `allowDragOnPress` is true, it passes `draggable: 'true'` to the pressable component via the pressProps, otherwise by default `draggable: 'false'` is provided. This would make most of our pressable components non-draggable by default and we could then make the drag handle button and the list/table row/cell elements draggable by passing `allowDragOnPress: true` to their respective usePress calls.

Then we could modify usePress's `shouldPreventDefault` check to find the target's closest draggable ancestor and the closest non-draggable ancestor and decide from there whether or not to prevent default on mousedown/pointerdown. If the target's closest draggable ancestor contains the closest non-draggable ancestor, we would prevent default. We would also prevent default if a draggable ancestor doesn't exist for the target. In every other case, we would not prevent default to allow drag and drop